build/cmake/find/FindQtLua.cmake

```diff
@@ -30,19 +30,17 @@ cmake_minimum_required(VERSION 3.0)
 # Find library - - tries to find *.a,*.so,*.dylib in paths hard-coded by the script
 
 find_library(QTLUA_LIBRARY
-   NAMES qtlua
-   PATHS /usr/lib /usr/local/lib /opt/lib /opt/local/lib /usr/local/qtlua/lib
-)
+			 NAMES qtlua
```

Comment:
Too much here as well. use only one additional tab when defining parameters of a functions in separate lines.